release-controller/tests/test_reconciler_integration.py

```diff
@@ -0,0 +1,285 @@
+import pydantic_yaml
+import typing
+import pytest_mock.plugin
+
+import const
+import commit_annotation
+import git_repo
+import dryrun
+from reconciler import Reconciler
+from reconciler_state import ReconcilerState
+from public_dashboard import DashboardAPI
+from dre_cli import ElectionProposal
+from release_index import Model as ReleaseIndexModel
+from release_index_loader import StaticReleaseLoader
+
+from tests.fixtures import ic_repo as ic_repo
+
+
+class AmnesiacReconcilerState(ReconcilerState):
```

Comment:
haha hilarious...
on a more serious note, how about calling it `TempdirReconsiderState` instead?